### ISSUE TITLE
ask the author of the AppImage to change the Categories in their .desktop file for better menu placement

### ISSUE THREAD

**Issue by snelweg** [realthunder/FreeCAD_assembly3#607]
Appimages work great, but I wonder why FreeCAD is always inserted into the 'other' menu on my Debian Duster install. Can I move it to a more appropriate place like 'development' or 'graphics'?

![appimageMenu](https://user-images.githubusercontent.com/1257901/117579172-09ec7e00-b0f2-11eb-8bd0-9ab15d6053bb.png)

I asked at the Appimage repo, they said:

`Not an AppImage issue. You need to ask the author of that AppImage to change the Categories in their .desktop file.
`
https://github.com/AppImage/AppImageKit/issues/1133 # 

So do you want to change it to a  more suiting menu category?

**Comment by realthunder** [comment]
Hmm... wonder if there is any way for user to set the category. I mean it's obviously not possible to choose a category that satisfy every one.

**Comment by snelweg** [comment]
> Hmm... wonder if there is any way for user to set the category. I mean it's obviously not possible to choose a category that satisfy every one.

No, according to the AppImage documentation the category is set when assembling the Image itself. I do agree it would be best if people could choose themselves.

**Comment by QuakeString** [comment]
I think "Graphics" could be the best category for FreeCAD

**Comment by realthunder** [comment]
FreeCAD.desktop list the category as `Engineering`, which is quite appropriate. So the problem boils down to how to add that category to your Linux distro.

**Comment by snelweg** [comment]
> FreeCAD.desktop list the category as `Engineering`, which is quite appropriate. So the problem boils down to how to add that category to your Linux distro.

Ok, will add that category to my menu's. Thanks for looking in to this.

**Comment by luzpaz** [comment]
Should we notate this somewhere ? Please advise

**Comment by snelweg** [comment]
> Should we notate this somewhere ? Please advise

What does that mean?

**Comment by luzpaz** [comment]
Add this to documentation so users can be aware of this possibility.
